### PR TITLE
fix: App crash when trying to change server backend in settings

### PR DESCRIPTION
**Pull request by trancenoid**
Fixes : #1819 

Changes: Changed default value of CUSTOM_SERVER from null to "", which is required since the default value must be empty (not null) when the app is used for the first time.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Please follow PR template

**Comment by trancenoid** [comment]
@iamareebjamal Sorry about that. Fixed it.

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 1]
```diff
@@ -259,7 +259,7 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
             flag = true
         }
         customerServer.isChecked = flag
-        inputUrlText.setText(PrefManager.getString(Constant.CUSTOM_SERVER, null))
+        inputUrlText.setText(PrefManager.getString(Constant.CUSTOM_SERVER, " "))
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Use "" instead of " ", it is faster

Comment (trancenoid) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Done!

**Pull request merged**